src/Polly/Timeout/TimeoutSyntax.cs

```diff
@@ -109,7 +109,9 @@ public static TimeoutPolicy Timeout(int seconds, TimeoutStrategy timeoutStrategy
     /// <param name="timeout">The timeout.</param>
     /// <returns>The policy instance.</returns>
     /// <exception cref="ArgumentOutOfRangeException">timeout;Value must be a positive TimeSpan (or Timeout.InfiniteTimeSpan to indicate no timeout).</exception>
+#pragma warning disable S3872
```

Comment:
The build failures are annoying - I've seen them before where dotnet format doesn't complain locally but does in CI. I think to fix them you need to move the disable to above the /// comments.

Reply:
Thanks, works like a charm now.